packages/twenty-server/src/engine/metadata-modules/index-metadata/index-metadata.entity.ts

```diff
@@ -13,6 +13,10 @@ import {
 import { IndexFieldMetadataEntity } from 'src/engine/metadata-modules/index-metadata/index-field-metadata.entity';
 import { ObjectMetadataEntity } from 'src/engine/metadata-modules/object-metadata/object-metadata.entity';
 
+export enum IndexType {
+  GIN = 'GIN',
```

Comment:
If we want to be exact, we should add BTREE here for "normal" indexes. Could also be the default value actually 🤔

Reply:
Added it !